packages/react-components/react-overflow/src/stories/Overflow/utils.stories.tsx

```diff
@@ -84,7 +84,8 @@ export const TestOverflowMenuItem: React.FC<TestOverflowMenuItemProps> = props =
     return null;
   }
 
-  return <MenuItem {...rest}>Item {id}</MenuItem>;
+  // As an union between button props and div props may be conflicting, casting is required
```

Comment:
does this mean that users that wrap the `MenuItem` component and spread props also need to cast, even if the type extends from `MenuItemProps` ?

Reply:
Yes! that is true for `Button` also, upper casting is required before spreading.